als-server/jvm/src/test/scala/org/mulesoft/als/server/lsp4j/Lsp4jLanguageServerImplTest.scala

```diff
@@ -67,7 +66,8 @@ class Lsp4jLanguageServerImplTest extends LanguageServerBaseTest with PlatformSe
     server.initialize(null).toScala.map(_ => succeed)
   }
 
-  test("Lsp4j LanguageServerImpl Command - Index Dialect") {
+  // todo: enable test after APIMF-3305 is adopted
+  ignore("Lsp4j LanguageServerImpl Command - Index Dialect") {
```

Comment:
This should be enabled? was enabled but lost it on a merge?
Jira is resolved, as the blocked one in ALS (ALS-1658)